benchmark/scatter.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const fork = require('child_process').fork;
+const path = require('path');
+const CLI = require('./_cli.js');
+
+//
+// Parse arguments
+//
+const cli = CLI(`usage: ./node benchmark/scatter.js <filename>
+  Run the benchmark script <filename> many times and output the rate (ops/s)
+  together with the benchmark variables as a csv.
+
+  --runs 30              number of samples
+  --var  variable=value  set benchmark variable (can be repeated)
+`, {
+  arrayArgs: ['var']
+});
+
+if (cli.required.length !== 1) {
+  cli.abort(cli.usage);
+  return;
+}
+
+// Create queue from the benchmarks list such both node versions are tested
+// `runs` amount of times each.
+const filepath = path.resolve(cli.required[0]);
+const name = filepath.slice(__dirname.length + 1);
+const runs = cli.optional.runs ? parseInt(cli.optional.runs, 10) : 30;
+
+let printHeader = true;
+
+(function recursive(i) {
+  const child = fork(path.resolve(__dirname, filepath), cli.optional.var);
+
+  child.on('message', function(data) {
+    // print csv header
+    if (printHeader) {
+      const confHeader = Object.keys(data.conf).join('", "');
+      console.log(`"filename", "${confHeader}", "rate", "time"`);
```

Comment:
Sorry, still confused. What's in the PR and what you have in your example look like they'll have different output to me. Mind showing me an example of what would be logged?